workflow/lib/sbs/find_peaks.py

```diff
@@ -1,6 +1,7 @@
 """Find SBS peaks."""
```

Comment:
Could you add more to this docstring? This may have originally been me :(

But now that find peaks has more code would be good to flesh this out